src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java

```diff
@@ -44,15 +44,15 @@ public void triggerOnScrollSlots() {
     jsActions.scrollToBottom();
   }
 
-  public void verifySlot(String slotName) {
+  public void verifySlot(String slotName, String slotElementId) {
```

Comment:
IMO you passing here too many arguments, we could guess `slotElementId` base on `slotName`, am I right? Or let's add additional `scrollToSlot` function with different input